FV3/wrapper/fv3gfs/wrapper/physics_properties.json

```diff
@@ -463,6 +463,14 @@
         "container": "Sfcprop",
         "dims": ["z_soil", "y", "x"]
     },
+    {
+        "name": "standard_deviation_of_subgrid_orography",
```

Comment:
This name should represent the entire array, not just the 0-index entry. It should probably be something like `orographic_variables`.